BUILD.md

```diff
@@ -398,6 +421,68 @@ Next, add the following lines to the application's `build.gradle` file:
     }
 ```
 
+**NOTE:** Make sure it is the app version of `build.gradle` (usually found in an
+app folder).
+This may not necessarily be the top `build.gradle` in the tree.
+
+##### Example: Adding the Capture Layer to the Vulkan Samples
```

Comment:
@bradgrantham-lunarg , I added a detailed section on building the layer into the Vulkan-Samples repo source.  I can remove this if you do not like it here.